src/sgame/sg_bot_util.cpp

```diff
@@ -1801,6 +1801,27 @@ void BotFireWeapon( weaponMode_t mode, usercmd_t *botCmdBuffer )
 		usercmdPressButton( botCmdBuffer->buttons, BTN_ATTACK3 );
 	}
 }
+
+static Cvar::Cvar<int> g_bot_upwardAttackMinHeight("g_bot_upwardAttackMinHeight", "minimal height difference for bots to attack upwards.", Cvar::NONE, 9999);
+
+// return true if an upward attack is started or in progress, false otherwise
+static bool BotAttackUpward( gentity_t *self )
```

Comment:
What is this supposed to do? I don't see anything about attacking in it, just moving.